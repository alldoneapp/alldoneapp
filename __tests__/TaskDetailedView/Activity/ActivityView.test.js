/**
 * @jest-environment jsdom
 */

import React from 'react'
import ActivityView from '../../../components/TaskDetailedView/Activity/ActivityView'

import renderer from 'react-test-renderer'

describe('ActivityView', () => {
    it('should render correctly', () => {
        const json = renderer.create(<ActivityView />).toJSON()
        expect(json).toMatchSnapshot()
    })
})