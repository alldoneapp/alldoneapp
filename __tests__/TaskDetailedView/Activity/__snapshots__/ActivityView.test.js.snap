// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityView should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "height": 200,
      "justifyContent": "center",
    }
  }
>
  <Text>
    ACTIVITY COMING SOON! ;)
  </Text>
</View>
`;
